exercism/secret-handshake/secret-handshake.spec.js

```diff
@@ -6,37 +6,37 @@ describe('Secret Handshake', function() {
     expect(handshake.commands()).toEqual(['wink']);
   });
 
-  xit('10 is a double blink', function() {
```

Comment:
when I looked it up, it appears that the xit() function skips the test? I assume you were working on this when the week ended :) Your code is looking good! I'd recommend putting up the code into github without the xit() even if the tests are failing.